src/main/org/codehaus/groovy/runtime/DefaultGroovyMethods.java

```diff
@@ -8928,6 +8932,27 @@ public int compare(Map.Entry<K, V> e1, Map.Entry<K, V> e2) {
         if (self.isEmpty()) {
             throw new NoSuchElementException("Cannot pop() an empty List");
         }
+        return self.remove(0);
+    }
+
+    /**
+     * Removes the last item from the List. Using add() and pop()
+     * is similar to push and pop on a Stack.
+     * <pre class="groovyTestCase">
+     * def list = ["a", false, 2]
+     * assert list.removeLast() == 2
+     * assert list == ["a", false]
+     * </pre>
+     *
+     * @param self a List
+     * @return the item removed from the List
+     * @throws NoSuchElementException if the list is empty and you try to pop() it.
```

Comment:
I think `if the list is empty` suffices here and don't think the rest of the sentence is needed (same for `pop` throws).